scripts/eslint-templates/src.eslintrc

```diff
@@ -2,8 +2,6 @@
 extends: airbnb
 parserOptions:
   sourceType: module
```

Comment:
eslint now has built in rules for import ordering so dropped plugin

Reply:
well, it allows grouping, not sorting but should allow sorting *soon*

https://github.com/benmosher/eslint-plugin-import/pull/1360